src/intrinsics/ecma262/Object.js

```diff
@@ -88,6 +88,11 @@ export default function(realm: Realm): NativeFunctionValue {
             throw new FatalError();
           }
 
+          to_must_be_partial = true;
+          frm.makeNotPartial();
```

Comment:
Maybe put a comment here that its safe to do this because we leak all of these objects.

Reply:
Seems like a comment will be helpful, just not that comment. ;-)

Maybe something like "Make this temporally not partial so that we can call frm.$OwnPropertyKeys below."

Reply:
To be clear, this code already existed, I just moved it around a bit.